backend/src/db/migrations/20230906103635_add_members_column_to_chats_table.ts

```diff
@@ -0,0 +1,27 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'chats';
+
+const ColumnName = {
+  MEMBERS: 'members',
```

Comment:
let's go with another approach and create users_to_chats table instead of members column, so that we can automatically delete user from chat if user was deleted

Reply:
So should this table look like this?
```mermaid
erDiagram
users_to_chats {
  int id PK
  dateTime created_at
  dateTime updated_at
  int user_id FK
  int chat_id FK
}
```

And I should delete the migration for members column then, right?

Reply:
yes